docs/_config.yml

```diff
@@ -7,16 +7,16 @@ kramdown:
 # The full version string
 # Used to match the version string portion of file paths, URLs, and dependency versions
 # e.g. client jar path,
-ALLUXIO_VERSION_STRING: 2.6.0-SNAPSHOT
+ALLUXIO_VERSION_STRING: 2.7.0-SNAPSHOT
 # We must inline the version string (e.g., "1.4.0-SNAPSHOT") rather than using the macro of Alluxio version.
 # Otherwise the macro name remains in the output.
-ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/alluxio-2.6.0-SNAPSHOT-client.jar
+ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/alluxio-2.7.0-SNAPSHOT-client.jar
 # For release branches, this should be the Alluxio version in the form of <major>.<minor>, e.g. both 1.5.0 and 1.5.0-RC1 should use "1.5"
 # For master branch, this should be "edge"
 # This should be used to reference versioned pages that are not relative to the docs/ directory
-# e.g. Javadoc:      https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION: 2.6
-# e.g. REST API doc: https://docs.alluxio.io/os/restdoc/{{site.ALLUXIO_MAJOR_VERSION: 2.6
-ALLUXIO_MAJOR_VERSION: 2.6
+# e.g. Javadoc:      https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/master/index.html
+# e.g. REST API doc: https://docs.alluxio.io/os/restdoc/{{site.ALLUXIO_MAJOR_VERSION}}/master/index.html
+ALLUXIO_MAJOR_VERSION: edge
```

Comment:
i think this needs to be 2.7 but not 100% certain. i would grep for `ALLUXIO_MAJOR_VERSION` in the docs folder to see how it's used to confirm

Reply:
I refer to this PR https://github.com/Alluxio/alluxio/pull/13011

Reply:
wait, in the previous PR that updated versions, it was already called `edge`. i think its okay then